scrypto-unit/src/test_runner.rs

```diff
@@ -438,6 +441,71 @@ impl<'s, S: ReadableSubstateStore + WriteableSubstateStore> TestRunner<'s, S> {
         let receipt = self.execute_manifest(manifest, vec![signer_public_key]);
         receipt.new_component_addresses[0]
     }
+
+    pub fn set_current_epoch(&mut self, epoch: u64) {
+        self.system_kernel_call(|kernel| {
+            kernel
+                .invoke_method(
+                    Receiver::Ref(RENodeId::System),
+                    FnIdentifier::Native(NativeFnIdentifier::System(SystemFnIdentifier::SetEpoch)),
+                    ScryptoValue::from_typed(&SystemSetEpochInput { epoch }),
+                )
+                .unwrap()
+        });
+    }
+
+    pub fn get_current_epoch(&mut self) -> u64 {
+        let current_epoch: ScryptoValue = self.system_kernel_call(|kernel| {
+            kernel
+                .invoke_method(
+                    Receiver::Ref(RENodeId::System),
+                    FnIdentifier::Native(NativeFnIdentifier::System(SystemFnIdentifier::SetEpoch)),
+                    ScryptoValue::from_typed(&SystemGetCurrentEpochInput {}),
```

Comment:
Should be `GetEpoch`

Reply:
Not to mention, should have tests which test `set_current_epoch` and `get_current_epoch`

Reply:
Sounds good, I will add that. The tests are likely to fail through until the issue with the `resim set-current-epoch` is fixed since they both use the same logic :)

Reply:
Fixed and added tests for that.